### PR TITLE
DOCS-#3609: Mention OmniSci engine in root README.md

### LINKED ISSUE #3609: Mention Omnisci in Readme.md
We need to add some words about Omnisci engine in Readme.md

### PR DESCRIPTION
**Pull request by Garra1980**
## What do these changes do?

- [x] commit message follows format outlined [here](https://modin.readthedocs.io/en/latest/contributing.html)
- [ ] passes `flake8 modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [ ] passes `black --check modin/ asv_bench/benchmarks scripts/doc_checker.py`
- [x] signed commit with `git commit -s` 
- [x]  Resolves #3609? 
- [ ] tests added and passing
- [ ] module layout described at `docs/developer/architecture.rst` is up-to-date

### CODE REVIEW

**Comment by Garra1980** [comment]
cc @aregm

File: README.md [hunk 1]
```diff
@@ -81,6 +98,8 @@ os.environ["MODIN_ENGINE"] = "dask"  # Modin will use Dask
 import modin.pandas as pd
 ```
 
+Check [complete Modin documentation](https://modin.readthedocs.io/en/latest/UsingOmnisci/index.html) for Omnisci engine setup
```
Comment (YarShev) [inline README.md:101]:
```suggestion
Check [this Modin section](https://modin.readthedocs.io/en/latest/UsingOmnisci/index.html) for Omnisci engine setup
```

**Review by YarShev — COMMENTED** [review]
LGTM, left a couple of suggestions.

File: README.md [hunk 2]
```diff
@@ -19,7 +19,9 @@ import modin.pandas as pd
 
 ### Installation
 
-Modin can be installed from PyPI:
+#### From PyPI
+
+Modin can be installed using following line:
```
Comment (devin-petersohn) [inline README.md:24]:
```suggestion
Modin can be installed with `pip`:
```

File: README.md [hunk 3]
```diff
@@ -37,6 +39,21 @@ pip install modin[all] # Install all of the above
 Modin will automatically detect which engine you have installed and use that for
 scheduling computation!
 
+#### From conda-forge
+
+Following line will install modin and 3 engines ([Ray](https://github.com/ray-project/ray),
```
[the remark this answers is not in the document]
Comment (devin-petersohn) [reply README.md:44]:
```suggestion
Installing from conda forge using `modin-all` will install Modin and 3 engines: ([Ray](https://github.com/ray-project/ray),
```

File: README.md [hunk 4]
```diff
@@ -37,6 +39,21 @@ pip install modin[all] # Install all of the above
...
+[Dask](https://github.com/dask/dask) and [Omnisci](https://modin.readthedocs.io/en/latest/UsingOmnisci/index.html))
+
+```bash
+conda install -c conda-forge modin-all
+```
+
+Also each engine could be installed explicitly. For example this line installs all 3:
```
Comment (devin-petersohn) [inline README.md:51]:
```suggestion
Each engine can also be installed individually:
```

File: README.md [hunk 5]
```diff
@@ -37,6 +39,21 @@ pip install modin[all] # Install all of the above
...
+
+```bash
+conda install -c conda-forge modin-ray modin-dask modin-omnisci
```
Comment (devin-petersohn) [inline README.md:54]:
```suggestion
conda install -c conda-forge modin-ray  # install Ray engine with Modin
conda install -c conda-forge modin-dask  # install Dask engine with Modin
conda install -c conda-forge modin-omnisci  # install Omnisci engine with Modin
```

**Review by devin-petersohn — COMMENTED** [review]
Looks pretty good, left some comments

**Comment by Garra1980** [comment]
Fixed all review comments

**Review by YarShev — APPROVED** [review]

**Review by devin-petersohn — APPROVED** [review]

**Pull request merged**